kotlinx-coroutines-core/jvm/test/channels/ConflatedBroadcastChannelNotifyStressTest.kt

```diff
@@ -5,10 +5,9 @@
 package kotlinx.coroutines.channels
 
 import kotlinx.coroutines.*
-import org.hamcrest.MatcherAssert.*
-import org.hamcrest.core.*
 import org.junit.*
 import java.util.concurrent.atomic.*
+import kotlin.test.assertEquals
```

Comment:
Start imports should be used (can be configured once, then do "optimize import" on files that were changed by commit)

Reply:
I've never heard of "start imports". Google suggests that it meant "starred imports". Did it guess right?

Reply:
Yes. That was type. I meant "star imports" as in `import kotlin.test.*`